src/acceptance-test/java/org/zalando/nakadi/webservice/EventStreamReadingAT.java

```diff
@@ -327,6 +334,52 @@ private Response readEvents() {
     }
 
     @Test(timeout = 10000)
+    public void whenExceedMaxConsumersNumThen429() throws IOException, InterruptedException, ExecutionException,
```

Comment:
I extended this test. Now it also checks if one slot is released when connection is closed.
https://github.com/zalando/nakadi/pull/489/commits/c8f0987d08e942a014fd0c2ebe7611fcc34a6ea2